models/database.go

```diff
@@ -263,13 +263,23 @@ var databaseSchema = [][]string{
 
 	13: {
 		`ALTER TABLE services
-			DROP CONSTRAINT address_socket_check`,
+			DROP CONSTRAINT address_socket_check;`,
```

Comment:
Not sure why we suddenly started to add `;`, but they are not required. I think it is better to remove them in all migrations for consistency.

Reply:
Removed.